bin/start

```diff
@@ -72,7 +74,7 @@ export DEV
 export SUMMON_ENV
 
 # summon environment variable
-export CONJUR_MAJOR_VERSION=4
+export CONJUR_MAJOR_VERSION=5
```

Comment:
We don't use this so I think this can be removed

Reply:
Summon will use it, and currently its useful for it to be explicitly set to 5.